data.py

```diff
@@ -0,0 +1,16 @@
+import csv
+from questions import Questions
+
+
+class Source:
+
+    @staticmethod
+    def from_csv(path: str = './data.csv') -> [Questions]:
+        questions = []
+        with open(path, 'r', encoding='utf-8') as f:
+            reader = csv.reader(f)
+            # headers
+            _ = next(reader, None)
+            for row in reader:
```

Comment:
`for row in csv:`

Reply:
It's duplicate with lib/csv.

what do you think about this?
```python
from csv import reader as csv_reader

...
csv = csv_reader(f)
next(csv, None)
...
for row in csv:
...
```